### PR TITLE
Update port-forward-access-application-cluster.md

### PR DESCRIPTION
**Pull request by baiyutang**
modify this, has two warning flag
image

### CODE REVIEW

File: content/zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 1]
```diff
@@ -227,13 +227,14 @@ local workstation to debug the database that is running in the pod.
 
 与本地 7000 端口建立的连接将转发到运行 Redis 服务器的 pod 的 6379 端口。通过此连接，您可以使用本地工作站来调试在 pod 中运行的数据库。
 
-{{< warning >}}
 <!--
+{{< warning >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:231]: This is not a correct fix. Please remove line 231.

The reason is that shortcode like `{{< warning >}}` in a HTML comment still works!!!

Comment (baiyutang) [reply content/zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:231]: ok

Comment (baiyutang) [reply content/zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md]: Done!

**Comment by tengqm** [comment]
/lgtm
/approve
Thanks.

**Pull request merged**